public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -141,8 +141,8 @@ export class SharedPreferences extends PureComponent<Props, State> {
         {() => {
           return (
             <>
-              <FieldSet label={<Trans i18nKey="shared-preferences.title">Preferences</Trans>} disabled={disabled}>
```

Comment:
I see that `<FieldSet ...` was never able to accept a disabled prop. Might be a bit of work, but if we add that through a React.Context that's provided by a `FieldSet`, we could have the disabled prop for children use that if it's not set on the child component props.

Reply:
I tried adding a storybook scenario for FieldSet and noticed that the child input is in fact disabled:

```tsx
export const Disabled: StoryFn<typeof FieldSet> = (args: Props) => {
  return (
    <Form onSubmit={() => console.log('Submit')}>
      {() => (
        <>
          <FieldSet disabled={true} {...args}>
            <Field label="Name">
              <Input name="name" />
            </Field>
          </FieldSet>
          <Button variant="primary" disabled={true}>Save</Button>
        </>
      )}
    </Form>
  )
}
```

After checking out how this happens, it's default DOM behavior to disable inputs in a `<fieldset disabled>`. Was there another reason that the child fields needed the `disabled` attr?